src/Orleans.CodeGeneration.Build/AssemblyResolver.cs

```diff
@@ -22,7 +22,8 @@ internal class AssemblyResolver
         /// Needs to be public so can be serialized accross the the app domain.
         /// </summary>
         public Dictionary<string, string> ReferenceAssemblyPaths { get; } = new Dictionary<string, string>();
-        
+
+        private readonly bool usedByCodeGenerator;
```

Comment:
is there a more semantically accurate name for this? eg: name it based on what it does rather than who it's for